### DISCUSSION TITLE
关于密码，放入.password后显示404

### DISCUSSION THREAD

**Discussion by danjiu** [spencerwooo/onedrive-vercel-index#614] [Q&A] [answered]
加密文档下放入.password 显示404 

说我未设置密码

{"error":"You didn't set a password."}

![1](https://user-images.githubusercontent.com/34980120/164261908-76c87b31-2b14-44b8-b4c7-b161697e2a72.png)
![2](https://user-images.githubusercontent.com/34980120/164261994-22889a06-980e-48f4-b343-eb174cf36721.png)

有什么解决办法吗

**Comment by spencerwooo** [comment, marked as answer]
API 有缓存机制，可能需要等一小时这边才能看到 `.password`，所以可以重新部署一下（Vercel 里面点 redeploy）强刷缓存。
